addons/mail/static/src/models/messaging_notification_handler/messaging_notification_handler.js

```diff
@@ -231,6 +232,13 @@ function factory(dependencies) {
                 return;
             }
 
+            if (!channel.isChatChannel && !isOdooFocused) {
+                this._notifyNewChannelMessageWhileOutOfFocus({
+                    channel,
+                    message,
+                });
+            }
+
```

Comment:
We don't want to call this method another time. We just need to make sure `!channel.isChatChannel` when calling it below. We still don't want to call it at all in the case `isChatWithOdooBot` for example.

Reply:
It was bugged anyway, got 2 notification.
Look like it's working now